tests/integration/components/pix-filter-banner-test.js

```diff
@@ -46,4 +48,37 @@ module('Integration | Component | filter-banner', function(hooks) {
     // then
     assert.equal(componentElement.textContent.trim(), '5 participants filtrés');
   });
+
+  test('it renders the PixFilterBanner with a FiltersLabel button text', async function(assert) {
+    //given
+    this.filterLabel = 'Effacer les filtres'
+    this.onClearFilters = sinon.spy();
+
+    // when
+    await render(hbs`
+      <PixFilterBanner @clearFiltersLabel={{filterLabel}} @onClearFilters={{this.onClearFilters}}>
+       content
+      </PixFilterBanner>
+    `);
+    const componentElement = this.element.querySelector('.pix-filter-banner__button');
+
+    // then
+    assert.equal(componentElement.textContent.trim(), this.filterLabel);
+  });
+
+  test('it should clear all filters when onClearFilters is triggered', async function(assert) {
+    // given
+    this.set('filters', 5);
+    
+    //when
+    await render(hbs`
+      <PixActionButton @onClearFilters={{this.onClearFilters}} />
+    `);
+    await click('button[type=button]');
+    this.onClearFilters = sinon.spy();
+    this.set('filters', 0);
+
+    // then
+    assert.equal(this.filters, 0);
+  });
```

Comment:
Ici on ne teste pas ce qu'on devrait tester : la seule chose qu'on peut tester, c'est si la fonction a bien été appelée :
```suggestion
  test('it should clear all filters when onClearFilters is triggered', async function(assert) {
    // given
    this.clearFiltersLabel = 'some label';
    this.onClearFilters = sinon.spy();
    
    //when
    await render(hbs`
      <PixActionButton @clearFiltersLabel={{clearFiltersLabel}} @onClearFilters={{this.onClearFilters}} />
    `);
    await click('button');

    // then
    assert.ok(this.onClearFilters.calledOnce, 'the callback should be called once');
  });
```

Reply:
Le test ne pass pas avec un simple  `await click('button'); ` car il arrive pas a detecter si le button a été appelé ou pas.
donc, il faut littéralement appeler la fonctionne dans le code. 
j'ai du faire  ' this. onClearFilters( ) ' pour passer le test, et même sans le await click('button').
C'est normale ça ?

Reply:
non c'est pas normal ^^ on peut regarder ça ensemble

Reply:
c'est bon , tous les tests passent !